doc/sphinx/reference/kinetics/reaction-rates.md

```diff
@@ -106,6 +106,72 @@ reactions:
 - [](sec-plog-rate)
 - [](sec-chebyshev-rate)
 
+
+(sec-electrochemical-reactions)=
+## Electrochemical Reactions
+
+In an electrochemical reaction (one that moves electrical charge from one phase of matter to another), the electric potential difference $\Delta\phi$ at the phase boundary exerts an additional "force" on the reaction that must be accounted for in the rate expression.
+
+The free energy of the reaction equals the electrochemical potential change, $\Delta\tilde{\mu}_{\rm rxn}$:
+
+$$\Delta\tilde{\mu}_{\rm rxn} = \Delta\mu_{\rm rxn} + n_{\rm elec}F\Delta\phi$$
+
+where $\mu_{\rm rxn}$ is the chemical potential, $n_{\rm elec}$ is the total electrical charge moved across the phase boundary (e.g. from "phase 1" to "phase 2", where $\Delta\phi = \phi_2 - \phi_1$), and $F$ is Faraday's constant (96,485 Coulombs per mole of charge).
+
+Cantera's charge transfer treatment assumes a reversible reaction with a linear energy profile in the region of the transition state.  From above, for any $\Delta\phi$ the free energy of the reaction changes by $n_{\rm elec}F\Delta\phi$.  The transition state energy, meanwhile, changes by a fraction of this, $\beta n_{\rm elec}F\Delta\phi$, where the "symmetry parameter" $\beta$ is a number between 0 and 1.
+
+This means that the activation energy for the reaction changes:
+- The barrier height for the forward reaction increases by $\beta n_{\rm elec}F\Delta\phi$.
+- The reverse reaction barrier height decreases by $\left(1-\beta\right) n_{\rm elec}F\Delta\phi$.
+
+Note that $n_{\rm elec}$ and $\Delta \phi$ both have a sign, so the terms "increase" and "decrease" are relative; the forward barrier height might increase by a negative amount (i.e. decrease), for instance.
+
+From transition state theory, the forward and reverse reaction rates are therefore calculated as:
+
+$$ R_f = k_f\exp\left(-\frac{\beta n_{\rm elec}F\Delta\phi}{RT} \right)\Pi_k C_{{\rm ac},\,k}^{\nu_k^\prime}\$$
+
+and
+
+$$ R_r = k_r\exp\left(\frac{\left(1-\beta\right)n_{\rm elec}F\Delta\phi}{RT} \right)\Pi_k C_{{\rm ac},\,k}^{\nu_k^{\prime\prime}},$$
+
+respectively, where $k_f$ and $k_r$ are the normal chemical rate coefficients in the absence of any electric potential difference (e.g., calculated using Arrhenius coefficients), $C_{{\rm ac},\,k}$ is the activity concentration of species $k$, $\nu_k^\prime$ and $\nu_k^{\prime\prime}$ are the forward and reverse stoichiometric coefficients, respectively, for species $k$ for this reaction, and $R$ and $T$ are the universal gas constant and temperature, respectively.
```

Comment:
"Activity concentration" is fairly unique bit of terminology used by Cantera. It's probably worth linking to the relevant C++ method, which you can do with the syntax ``` {ct}`activity concentration <ThermoPhase::getActivityConcentrations>` ```

Also, I think I'd say "reactant" and "product" stoichiometric coefficients rather than forward/reverse.

Reply:
Good point, re: activity concentration. I wish it were less unique to Cantera, but alas :) 

I personally prefer forward and reverse, but will defer to your judgement here (which is I think more consistent with how Cantera actually does the calculations, anyway).